SConstruct

```diff
@@ -539,6 +539,9 @@ if selected_platform in platform_list:
                 env.module_icons_paths.append(path + "/" + "icons")
             modules_enabled[name] = path
 
+            if getattr(config, "has_custom_iterator", False) and config.has_custom_iterator():
+                env.Append(CPPDEFINES=["CUSTOM_ITERATOR"])
```

Comment:
Some other module may already define this. Not sure if crucial but shouldn't hurt anything.
```suggestion
                env.AppendUnique(CPPDEFINES=["CUSTOM_ITERATOR"])
```

Reply:
I can add it, yes.